### PR TITLE
Some patches

### PR DESCRIPTION
**Pull request by user202729**
Apart from bug fixes, I also add a few paragraphs which you may want to verify carefully before merging (and a few *comments* that... should probably be addressed)

### CODE REVIEW

**Comment by vEnhance** [comment]
Thanks much! Will take a closer look sometime within the next week.

**Comment by vEnhance** [comment]
> 		% TODO must be Laurent series? But I think not all, only count those convergent in a positive radius around center?

Why does convergence matter here?

**Comment by vEnhance** [comment]
Other than that one question from me I think all the other suggestions are really nice, thanks for this. :)

**Comment by vEnhance** [comment]
I'm going to merge this now just so I don't forget later (still on the road, so my to-do queue is still a mess) but if there are more things that need to be addressed feel free to open another issue/PR or just comment here.

**Comment by user202729** [comment]
> > ```
> >   % TODO must be Laurent series? But I think not all, only count those convergent in a positive radius around center?
> > ```
> 
> Why does convergence matter here?

Actually I forgot. But I think what I was trying to say is that...
* the set A_𝔪 is not exactly the same as (read: isomorphic to as rings) the set of Laurent series
* in particular, any element of the form f(x)/g(x) can only blow up at 𝒱(g), which is a closed subset of 𝔸¹ that is not 𝔸¹ itself, so it's a union of finitely many points apart from 0

but yes in retrospect, apart from that issue, "Laurent series" also include the functions that blow up at 0 such as 1/x, which should not be in the set A_𝔪. (it would be A_𝔪[1/x] which I think is equal to the fraction field of the local ring A_𝔪...? I'm not too sure about this)

File: tex/alg-NT/dedekind.tex [hunk 1]
```diff
@@ -526,6 +536,13 @@ \section{Fractional ideals}
 			\mid x\ka \subseteq (1) = \mathcal A \right\} \]
 	is a fractional ideal with $\ka \ka\inv = (1)$.
 \end{theorem}
+
+(This result is nontrivial.
```
Comment (user202729) [inline tex/alg-NT/dedekind.tex:540]: image

In retrospect, it's already mentioned somewhere else in the book.

**Pull request merged**